app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -408,6 +409,33 @@ def test_update_experiment_is_rollout(self):
         experiment = NimbusExperiment.objects.get(id=experiment_id)
         self.assertTrue(experiment.is_rollout)
 
+    def test_update_live_rollout(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            NimbusExperimentFactory.Lifecycles.LAUNCH_APPROVE_APPROVE,
+            application=NimbusExperiment.Application.FENIX,
+            is_rollout=True,
+            population_percent=40,
+        )
+        experiment_id = experiment.id
+        response = self.query(
+            UPDATE_EXPERIMENT_MUTATION,
+            variables={
+                "input": {
+                    "id": experiment.id,
+                    "isRollout": True,
```

Comment:
This field won't get set on the audience page (it's set on the branches page I think) so I'd leave it out of this particular test to more closely resemble the query that will come from the audience page?